server/routes/viewingLicences/handlers/viewCases.test.ts

```diff
@@ -1748,7 +1735,7 @@ describe('Route handlers - View and print case list', () => {
       })
 
       it('should return a previously approved licence', async () => {
-        const caseList = new Container([
```

Comment:
no longer using this Container class in a load of places - it wasn't providing enough value for it's weight